### ISSUE TITLE
Unable to rewrite .app file

### ISSUE THREAD

**Issue by mdix** [aeternity/testnet#127]
When trying to build the latest version (5a4c4545d3fe8c8ee7cbf6006c5c71559d55713f) via `make prod-release` I get the following error:

```
===> Verifying dependencies...
===> Compiling ae_api
===> Compiling ae_core
===> Starting relx build process ...
===> Resolving OTP Applications from directories:
          /Users/marcdix/crypto/aeternity/testnet/_build/prod/lib
          /Users/marcdix/crypto/aeternity/testnet/apps
          /usr/local/Cellar/erlang/19.3/lib/erlang/lib
          /Users/marcdix/crypto/aeternity/testnet/_build/prod/rel
===> Resolved ae_core-0.1.0
===> Unable to rewrite .app file /Users/marcdix/crypto/aeternity/testnet/_build/prod/rel/ae_core/lib/public_key-1.4/ebin/public_key.app due to {error,
                                                                                                                                           eacces}
make: *** [prod-release] Error 127
```

OS: MacOS 10.12.5
Erlang: Erlang/OTP 19 [erts-8.3] [source] [64-bit] [smp:8:8] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]

Any help appreciated.

**Comment by beaugunderson** [comment]
what's the output of `ls -al /Users/marcdix/crypto/aeternity/testnet/_build/prod/rel/ae_core/lib/public_key-1.4/ebin/public_key.app`?

**Comment by zack-bitcoin** [comment]
You are using an old version of the software.
It wasn't working with mac for a short time.
The current version should work with Mac.

**Comment by mdix** [comment]
Ok, so I now updated to `73ac6d9db9eb42ee712c9521bc1ab530f4c51246` and I still have the same issue. 

`make release-build`

Outputs:
```
sed -e "\
    s:%% comment:\
    {port, 8040},\
    {peers, [[{46,101,103,165}, 8080]]},\
    {master_pub, <<\"BO8I1h5yIliI8XPCT89TMTqWvsmZ0J0D13cwF8UZ9YrL2oIdKZUAVg2L100okp1wtYCecxPC8kyPigBMC/lvg1Y=\">>},\
    {test_mode,false},\
    :\
    " config/sys.config > config/prod/sys.config
===> Verifying dependencies...
===> Compiling ae_api
===> Compiling ae_core
===> Starting relx build process ...
===> Resolving OTP Applications from directories:
          /Users/marcdix/crypto/aeternity/testnet/_build/prod/lib
          /Users/marcdix/crypto/aeternity/testnet/apps
          /usr/local/Cellar/erlang/19.3/lib/erlang/lib
          /Users/marcdix/crypto/aeternity/testnet/_build/prod/rel
===> Resolved ae_core-0.1.0
===> Unable to rewrite .app file /Users/marcdix/crypto/aeternity/testnet/_build/prod/rel/ae_core/lib/public_key-1.4/ebin/public_key.app due to {error,
                                                                                                                                           eacces}
make: *** [release-build] Error 127
```
